src/connection_wrap.h

```diff
@@ -0,0 +1,37 @@
+#ifndef SRC_CONNECTION_WRAP_H_
+#define SRC_CONNECTION_WRAP_H_
+
+#if defined(NODE_WANT_INTERNALS) && NODE_WANT_INTERNALS
+
+#include "env.h"
+#include "stream_wrap.h"
+#include "v8.h"
+
+namespace node {
+
+template <typename WrapType, typename UVType>
+class ConnectionWrap : public StreamWrap {
+ public:
+  UVType* UVHandle() {
+    return &handle_;
+  }
+
+  static void OnConnection(uv_stream_t* handle, int status);
+
+ protected:
+  ConnectionWrap(Environment* env,
+                 v8::Local<v8::Object> object,
+                 ProviderType provider,
+                 AsyncWrap* parent);
+  ~ConnectionWrap() {
+  }
```

Comment:
If you are leaving the constructor empty, you might as well make it `~ConnectionWrap() = default;`